components/Footer.js

```diff
@@ -1,23 +1,36 @@
 import styled from "styled-components";
 
-export default function Footer({ handleNav }) {
+export default function Footer({ handleUser, handleCostumer, handleStart }) {
   return (
     <StyledFooter>
-      <button onClick={handleNav}>User</button>
+      <button onClick={handleUser}>User</button>
+      <button onClick={handleStart}>Home</button>
+      <button onClick={handleCostumer}>Costumer</button>
```

Comment:
```suggestion
      <button onClick={onUser}>User</button>
      <button onClick={onStart}>Home</button>
      <button onClick={onCostumer}>Costumer</button>
```
Again, mazbe you can rename Customer and User, so its more clear what is happening here :)